### DISCUSSION TITLE
Thanks for continual improvement!

### DISCUSSION THREAD

**Discussion by Kitenite** [haotian-liu/LLaVA#317] [General]
Just want to thank the devs and researchers for the continual improvements since release. Unlike a lot of other research projects that don't continue improving on the codebase, I love seeing the updates on Llava to continue being state-of-the-art. Kudos!

**Comment by irfantogluk** [comment]
A thanks from me too with ❤️

**Comment by haotian-liu** [comment]
Thanks for the nice words and support on this :)

Let's work together to continue pushing the SOTA 🎉

**Comment by Don-Chad** [comment]
Many thanks for sharing this and keeping up the improvement!
